source/userguide/optional_dependency.rst

```diff
@@ -0,0 +1,30 @@
+Managing Optional Dependency
+============================
+
+MindMeld depends on Duckling for numerical parsing and ElasticSearch for Entity Resolution and Question Answering. However if your application does not need these components, you can turn off them by configuring your MindMeld application.
```

Comment:
does not need these components -> does not need the fully optimized version of these components 

you can use Entity Recognition without ES, but it is a very basic resolver